rwengine/src/render/ObjectRenderer.cpp

```diff
@@ -308,7 +308,8 @@ void ObjectRenderer::renderVehicle(VehicleObject* vehicle,
 void ObjectRenderer::renderPickup(PickupObject* pickup, RenderList& outList) {
     if (!pickup->isEnabled()) return;
 
-    glm::mat4 modelMatrix = glm::translate(glm::mat4(1.0f), pickup->getPosition());
+    glm::mat4 modelMatrix =
+        glm::translate(glm::mat4(1.0f), pickup->getPosition());
```

Comment:
Looks like there's just formatting changes in this file

Reply:
getVisible was placed with isVisible